### DISCUSSION TITLE
v0.21.4 - Import BlocksManager, PluginManager, RichtextEditor

### DISCUSSION THREAD

**Discussion by Wayne-Mather** [GrapesJS/grapesjs#5299] [Q&A]
Hi,

Seems I can no longer import the BlocksManager, PluginManager and RichTextEditor?

We have moved to the new import type { XXX } from 'grapesjs'; 

But for some reason these exported classes cannot be imported so we cannot define the type on our functions. Besides moving to any or @ts-ignore - Can anyone inform me how to import these types?

Thanks in advance for any assistance.

![image](https://github.com/GrapesJS/grapesjs/assets/11050269/efc6e46c-8b0f-44e8-8c3f-dc5076175dfc)

**Comment by artf** [comment]
> Seems I can no longer import the BlocksManager, PluginManager and RichTextEditor?

I don't think it was ever possible, but you can still get them from the `Editor`, eg. `BlockManager: Editor['Blocks']`
